pkg/graph.go

```diff
@@ -59,7 +59,8 @@ func NewGraph(dirFS fs.FS) (*Graph, error) {
 			}
 			name := c.Package.Name
 			if name == "" {
-				log.Fatalf("no package name in %q", path)
+				log.Printf("no package name in %q", path)
```

Comment:
nit: (just to be explicit about the effect this circumstance will have on the DAG) 🤷 

```suggestion
				log.Printf("no package name in %q, skipping...", path)
```